org.eclipse.scout.rt/pom.xml

```diff
@@ -123,19 +123,19 @@
     <scout.base.version>25.1.0</scout.base.version>
     <base.version>${scout.base.version}</base.version>
     <org.eclipse.scout.rt.version>${project.version}</org.eclipse.scout.rt.version>
-    <jetty.version>11.0.24</jetty.version>
+    <jetty.version>12.0.15</jetty.version>
```

Comment:
12.0.16 has been released last week, do we want to upgrade to the latest and greatest?